_posts/2018-05-17-self-organization.md

```diff
@@ -0,0 +1,43 @@
+
+* Backlog (One for each category)
+* Planned for This Week
+* Grooming
+* Ready To Do
+* In Progress
+* Done Weekly
+* Done Overall
+
+Each step has its own unique purpose. When I find a new subject to practice, I add it to the backlog. On each Monday, I have a planning meeting with my mentors, in that meeting, we pick topics to cover in the following week. I move these tasks to **Planned for This Week** column afterwards. After the meeting, I evaluate the tasks if they are small and descriptive enough. If not, big tasks go to **Grooming** column. In the **Grooming** column, I either investigate about the task or hack it quickly to understand if I can create smaller and descriptive pieces from that single big task. These new, small and descriptive tasks go to the **Ready To Do** column. In the **In Progress** column, I keep the task I am currently working on. During the day, you can check anytime to see what I am doing, that's what I call transparency. Ideally, there should be only one task active in the **In Progress** column. After I complete a task, I move them to **Done Weekly** column. Tasks wait here to be reviewed on the next planning meeting. After the weekly meeting, they are moved to **Done Overall** column. This column is analysed by the visualization tool, I will come to that.
+
+It's extremely important for me to have small and descriptive tasks. I can easily forget what exactly I have done during an enormous task. If tasks are descriptive enough, they will reflect the information right away. If they're small enough, I can feel the power of the accomplishment during the day. It's also important for me to track my learnings in the future, on the other hand, it could be important for the next apprentices to understand my 6 months.
+
+I've labelled each task with respect to the seven categories. These labels helped me to distinguish each task programmatically, as a follow-up step, I've implemented a basic data visualization application for my data in the board. It's live in [here](https://gracious-booth-c323f1.netlify.com/) and you can easily see my distribution amongst the categories. While preparing my apprenticeship half-time presentation, this application helped me more than I've expected.
+
+I've calculated **# of working days** during my apprenticeship and the **# of tasks I've completed**. When I divided these numbers, it turned out I have completed approximately three tasks per day. I wanted to be more specific about my distribution amongst the categories and I've ended up with the following numbers, by dividing **# of completed tasks in the category** to three;
+
+* Software Design; 20 Days
+* Learning; 16 Days
+* Communication; 14 Days
+* Testing; 9.3 Days
+* Tools; 9 Days
+* Agile Methods; 5.6 Days
+* Others; 2.6 Days
+
+By sticking to the plan, I've managed to reflect what I've focused on during my apprenticeship.
```

Comment:
I'm missing a conclusion to the post, something that wraps it up. Here's an idea on what it could look like: 

"By sticking to the plan, I've managed to reflect on what my focus during the apprenticeship has been. I find this to be just as valuable as the actual learnings, since it allows me to see how far I've come, and just how useful this process has been to my growth as a Software Crafter."

Or... you know, whatever note you want to finish on 😅.